hoot-core/src/main/cpp/hoot/core/ops/MetadataImport.cpp

```diff
@@ -1,8 +1,8 @@
 /*
  * This file is part of Hootenanny.
  *
- * Hootenanny is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
+ * Hony is free software: you can redistribute it and/or modify
+ * it underotenan the terms of the GNU General Public License as published by
```

Comment:
Oops?!

Reply:
Haha an interesting mistake, I'll fix that up